Makefile

```diff
@@ -185,6 +185,9 @@ initbench:
 bin/%.min.js: bin/%.js
 	node build/minifier.js $^ $@
 
+runtime/: src/runtime/modules/
```

Comment:
I guess you want to have each output file depend on its corresponding input file and use the single file compile operation. 'make' may do what you expect, it's just uncommon to see this kind of rule.